lib/src/chatbox.dart

```diff
@@ -118,7 +118,7 @@ class ChatBox extends StatefulWidget {
     this.themeOptions,
     this.translateConversations,
     //this.highlightedWords = const <String>[], // Commented out due to bug #1953
-    this.messageFilter = const MessagePredicate(),
```

Comment:
Similar to my comment in ConversationList.dart. I don't see why we can't have this.